### PR TITLE
Removed ambiguity of term Mesos Containerizer.

### PR DESCRIPTION
**Pull request by joerg84**
In Mesos terms the Mesos Containerizer is one specific containerizer (http://mesos.apache.org/documentation/latest/containerizer/).

## Urgency
- [ ] Blocker 
- [ ] High
- [ ] Medium

## Requirements
- Test all commands and procedures.
- Build content [locally](https://github.com/dcos/dcos-docs#test-local) and test for formatting/links.
- Add redirects to [dcos-website/redirect-files](https://github.com/dcos/dcos-website#managing-redirects).
- Change all affected versions (e.g. 1.7, 1.8, and 1.9).
- See the [contribution guidelines](https://github.com/dcos/dcos-docs#contributing).

### CODE REVIEW

File: 1.9/overview/concepts.md [hunk 1]
```diff
@@ -348,7 +348,7 @@ A Mesos resource offer provides a set of unallocated resources (e.g. cpu, disk,
 
 ### <a name="mesos-containerizer"></a>Containerizer
```
Comment (Gilbert88) [inline 1.9/overview/concepts.md]: We may also want to fix this subtitle as well. Could we follow a commit to fix it?

Comment (Gilbert88) [inline 1.9/overview/concepts.md]: We may also want to fix this subtitle as well. Could we follow a commit to fix it?

Comment (joerg84) [inline 1.9/overview/concepts.md]: The subtitle imo is prefixed by mesos- as it is mesos specific terms...

Comment (Gilbert88) [inline 1.9/overview/concepts.md]: I mean s/mesos-containerizer/containerizer/g

**Comment by Gilbert88** [comment]
Ship it!

File: 1.9/overview/concepts.md (hunk 1 above)
Comment (joerg84) [inline 1.9/overview/concepts.md]: If you look at the list of concepts https://github.com/dcos/dcos-docs/pull/929/files#diff-2050705536f314fc0acd1315e4caaf06L267 they are all prefixed by mesos- (as they are mesos concepts).

**Pull request merged**